tests/wpt/metadata/webaudio/the-audio-api/the-audioparam-interface/audioparam-setValueCurveAtTime.html.ini

```diff
@@ -1,2 +1,124 @@
 [audioparam-setValueCurveAtTime.html]
-  expected: ERROR
+  [X Max error for test 12 at offset 15876 is not less than or equal to 0.0000037194. Got 0.4649144411087036.]
+    expected: FAIL
```

Comment:
Most of these tests shouldn't be failing. Are you sure the servo-media code is doing the right thing?

Reply:
I was also confused about the same. We get the expected output while running the code from the servo/media directory as I printed out the values and time to verify. But I got similar amount of failures when I ran the tests for linearRampToValueAtTime, exponentialRampToValueAtTime etc so I thought this was common. 

I also noticed that a lot of error were of the following type: 
 [X Discontinuity at index is not equal to 7938. Got 7940.]
    expected: FAIL

  [X Discontinuity at index is not equal to 6615. Got 6617.]
    expected: FAIL

  [X Discontinuity at index is not equal to 11907. Got 11909.]
    expected: FAIL

  [X Discontinuity at index is not equal to 9261. Got 9263.]
    expected: FAIL
There is a mismatch by 2 for all of these. 

Is there somewhere I can look to fix these or is this something I don't have to look into?

Reply:
This could be a timing synchronization bug which wouldn't be your fault.

You're right that most seem minor, though the Max error ones don't seem so minor.

Reply:
Filed https://github.com/servo/servo/issues/23271, ignore this issue for now.